pymc3/tests/test_types.py

```diff
@@ -38,26 +38,44 @@ def teardown_method(self):
 
     @change_flags({'floatX': 'float64', 'warn_float64': 'ignore'})
     def test_float64(self):
+        data = np.random.randn(5)
+
+        with Model() as coarse_model:
```

Comment:
instead of mixing these tests - could you just make a copy of it for mlda? If not, please don't create the `coarse_model` when it's not used (worried about test runtime)

Also for the tests below...

Reply:
Separated MLDA tests into two new functions within the `TestType` class